fedot/core/operations/evaluation/operation_implementations/models/ensemble/blending.py

```diff
@@ -0,0 +1,160 @@
+from typing import Optional
+
+import numpy as np
+import optuna
+from sklearn.metrics import log_loss, mean_squared_error
+
+from golem.core.log import default_log
+
+from fedot.core.data.data import InputData, OutputData
+from fedot.core.operations.evaluation. \
+    operation_implementations.implementation_interfaces import ModelImplementation
+from fedot.core.operations.operation_parameters import OperationParameters
+from fedot.core.repository.tasks import TaskTypesEnum
+
+
+class BlendingImplementation(ModelImplementation):
+    """Base class for weighted average blending."""
+
+    def __init__(self, params: Optional[OperationParameters] = None):
+        super().__init__(params)
+        self.seed = 42  # constant seed for optuna
```

Comment:
nit: даже если это внутренний seed для `TPESampler`, хочется иметь возможность его передавать в конструкторе в виде параметра

```suggestion
        self.seed = params.get('seed', 42)  # constant seed for optuna
```